flask_frontend/app.py

```
@@ -0,0 +1,49 @@
+def index():
+    '''
+    Render the home page and handle project submission.
+
+    Returns:
+        str: Rendered HTML for the page.
+    '''
+    if request.method == 'POST':
+        repo_url = request.form.get('repo_url', '').strip()
+        repo_type = request.form.get('repo_type', 'github')
+        token = request.form.get('token', '').strip()
+        params = {
```

Ruff (0.11.9)

Local variable `params` is assigned to but never used

Remove assignment to unused variable `params`

(F841)